e2e/test/scenarios/dashboard-filters/reproductions/22788-flter-cc-dropped-on-second-edit.cy.spec.js

```diff
@@ -10,6 +10,7 @@ import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
 const { PRODUCTS_ID, PRODUCTS } = SAMPLE_DATABASE;
 
 const ccName = "Custom Category";
+const ccDisplayName = "Product.Custom Category";
```

Comment:
@nemanjaglumac this is how it looks on Cypress. I need to modify the assertion because the name doesn't exactly match.
<img width="228" alt="image" src="https://user-images.githubusercontent.com/1937582/222739810-4b220efb-5c94-4943-86fd-bc9aa37458ab.png">

Reply:
Great catch, @WiNloSt !

Nit:
it's not really a display name, as much as "columnMappedToCC", but it doesn't make a huge difference here. Please do not change it now and trigger another CI run.